csaf-schema/src/test/resources/csaf-strict-test.json

```diff
@@ -0,0 +1,215 @@
+{
+  "this_property_does_not_exist": {},
+  "document": {
+    "aggregate_severity": {
+      "text": "Moderate"
+    },
+    "category": "csaf_security_advisory",
+    "csaf_version": "2.0",
+    "distribution": {
+      "tlp": {
+        "label": "WHITE",
+        "url": "https://www.first.org/tlp/"
+      }
+    },
+    "lang": "en-US",
+    "publisher": {
+      "category": "coordinator",
+      "name": "Bundesamt für Sicherheit in der Informationstechnik",
+      "namespace": "https://www.bsi.bund.de"
+    },
+    "title": "CVRF-CSAF-Converter: XML External Entities Vulnerability",
+    "tracking": {
+      "current_release_date": "2022-03-17T13:03:42.105Z",
+      "generator": {
+        "date": "2022-03-17T13:09:42.105Z",
+        "engine": {
+          "name": "Secvisogram",
+          "version": "1.12.1"
+        }
+      },
+      "id": "BSI-2022-0001",
```

Comment:
This applies also to the `/document/tracking/id` from which the filename MUST be determined. So for the given filename:
```suggestion
      "id": "CSAF-Strict-Test",